src/test/java/de/bonndan/nivio/input/LandscapeDescriptionFactoryTest.java

```diff
@@ -144,15 +146,13 @@ public void readEnvVars() throws IOException, NoSuchFieldException, IllegalAcces
             if ("java.util.Collections$UnmodifiableMap".equals(c.getName())) {
                 Field m = c.getDeclaredField("m");
                 m.setAccessible(true);
-                var x = (Map<String, String>) m.get(System.getenv());
-                x.put("PRIVATE_TOKEN", "veryPrivateToken");
```

Comment:
In my Windows 10 system, this is not able to modify the environment variables of the OS (although I see PRIVATE_TOKEN in a watcher while debugging in System.getenv()).

What this test attempts to do is to verify that an env from the e.g. `example_environment_vars.yml` is read by the `LandscapeDescriptionFactory`, correct?

So reading e.g. just `JAVA_HOME` (which should be set on all OS we are using), will do the trick too? @bonndan : what do you think?

Reply:
Yes, this is the purpose of the test and yes, reading JAVA_HOME would demonstrate that this env var can be read. Do you have time to change the test?

Reply:
btw: the docker-compose stuff is outdated and left there for demo purposes. Since you tried to run it, something must be unclear in the documentation. Could you open an issue explaining the problem?